python/lsst/pipe/supertask/activator.py

```diff
@@ -292,7 +321,11 @@ def load_super_task(super_taskname):
 
         if super_task_module:
             # Need to add a try -- except here
-            python_mod_stask = importlib.import_module(package.__name__ + '.' + super_task_module)
+            if hasattr(package, 'mock'):
+                (file_path, filename, data) = imp.find_module(super_task_module, package.__path__)
+                python_mod_stask = imp.load_module(super_task_module, file_path, filename, data)
```

Comment:
I have been on and off with the imp module. I removed and then added back for the manual path modules. I started by appending the path to sys.path[0] but I had doubts whether that might mess things up if there is a conflict name or any race conditions. Is there any other alternative that will be compatible with Py2/3.? If not, I'll use sys.path